suave/e2e/pebble_test.go

```diff
@@ -0,0 +1,79 @@
+package e2e
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/suave/sdk"
+	"github.com/stretchr/testify/require"
+)
+
+func WithPebbleDB(t *testing.T) frameworkOpt {
+	tmpDir := t.TempDir()
+	return func(c *frameworkConfig) {
+		c.suaveConfig.PebbleDbPath = tmpDir
+	}
+}
+
+func TestPebbleStore(t *testing.T) {
```

Comment:
There is a [generic test for the backends](https://github.com/flashbots/suave-geth/blob/main/suave/cstore/redis_store_backend_test.go#L7) so that we do not have to rely on `e2e` to cover them. Can you use that instead?